www/src/utils/node/docs.js

```diff
@@ -24,6 +26,42 @@ const docSlugFromPath = parsedFilePath => {
   }
 }
 
+/*
+ * Blog posts are occassionally set to the future (e.g. publish on DD/YY/MMMM)
+ * We'll filter as part of the build to only show posts for _today_ or in the past
+ * Note: we only use this behavior in production
+ */
+const isReleased = node => {
+  if (IS_PRODUCTION) {
+    let released = false
+    const date = _.get(node, `frontmatter.date`)
+    if (date) {
+      released = moment.utc().isSameOrAfter(moment.utc(date))
+    }
+    return released && node.draft !== true
+  }
+  return true
+}
+
+exports.createSchemaCustomization = ({ actions, schema }) => {
```

Comment:
This will have a perf hit, because we need to do a pass on all blog posts before we get this info. A better way to do it would be to add this data in onCreateNode or when creating blog posts.

Reply:
(This already was on onCreateNode 🙃 -- that isn't re-invoked when blog posts are cached)